public/app/plugins/panel/text/models.gen.ts

```diff
@@ -1,24 +1,22 @@
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-// NOTE: This file will be auto generated from models.cue
-// It is currenty hand written but will serve as the target for cuetsy
+// This file was autogenerated by cuetsy. DO NOT EDIT!
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-export const modelVersion = Object.freeze([1, 0]);
+export const modelVersion = Object.freeze([0, 0]);
 
 export enum TextMode {
   HTML = 'html',
   Markdown = 'markdown',
 }
 
 export interface PanelOptions {
-  mode: TextMode;
   content: string;
+  mode: string;
```

Comment:
This looks a bit off, shouldn't this be `TextMode`
```suggestion
  mode: TextMode;
```

Reply:
hmm, thought i'd fixed this one before last night's last push, apparently not